index.css

```diff
@@ -0,0 +1,68 @@
+@import url("https://argon-css.netlify.app/components/index.css");
+
+/* header search bar */
+
+.search__bar__container {
+  width: 30rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
As making div center is a common use case you can use a combined utility class. Totally a suggestion. Not an improvement.

Reply:
okay, i already have a clasd for that, will use it here